### PR TITLE
Support encrypted ssh (passhrase protected keys)

### PR DESCRIPTION
**Pull request by cice**

### CODE REVIEW

**Comment by cice** [comment]
#1

**Comment by xetys** [comment]
Super cool, I'll check out that today

File: cmd/util.go [hunk 1]
```diff
@@ -11,22 +11,112 @@ import (
...
 
-func runCmd(node Node, command string) (output string, err error) {
-	index, privateKey := AppConf.Config.FindSSHKeyByName(node.SSHKeyName)
+var sshPassPhrases = make(map[string][]byte)
+
+func capturePassphrase(sshKeyName string) (error) {
+	index, privateKey := AppConf.Config.FindSSHKeyByName(sshKeyName)
 	if index < 0 {
-		return "", errors.New(fmt.Sprintf("cound not find SSH key '%s'", node.SSHKeyName))
+		return errors.New(fmt.Sprintf("cound not find SSH key '%s'", sshKeyName))
```
Comment (eliasp) [inline cmd/util.go]: Typo (`cound` → `couldn't`)

**Comment by xetys** [comment]
fixed some little stuff and now LGTM!

**Pull request merged**